common/src/ticker.rs

```diff
@@ -0,0 +1,35 @@
+use crate::Step;
+use serde::{Deserialize, Serialize};
+
+#[derive(Debug, Serialize, Deserialize)]
+pub struct TickerEntity {
+    pub last_ticked: Step,
```

Comment:
This parameter is currently unused. If you get rid of it, you can get rid of this wrapper entirely and make `Ticker` the component instead.

Reply:
Done!